src/content/project/ios-base.md

```diff
@@ -2,7 +2,7 @@
 title: iOS-base
 author: OpenBytes
 pubDatetime: 2022-12-01T12:00:00Z
-postSlug: ios-base
+projectSlug: ioS-base
```

Comment:
ios or iOS

Reply:
OOOOF LOL

Reply:
Fixed on this commit [664dcc9](https://github.com/0xOpenBytes/Site/pull/18/commits/664dcc951a265182fb25f6ed01bfd6d95a958faf)